internal/cmd/skupper/token/kube/token_issue.go

```diff
@@ -101,7 +101,7 @@ func (cmd *CmdTokenIssue) ValidateInput(args []string) error {
 		if !ok {
 			validationErrors = append(validationErrors, fmt.Errorf("there is no active skupper site in this namespace"))
 		} else {
-			ok, siteName = utils.SiteLinkAccessEnabled(siteList, common.LinkAccessTypes)
```

Comment:
If the func 
`SiteLinkAccessEnabled(siteList *v2alpha1.SiteList, linkAccessTypes []string) (bool, string)
is not used anywhere else, could it be removed?

(same goes for `SiteConfigured(siteList *v2alpha1.SiteList) bool`, not related to this pr, but it is in the same file).

Reply:
makes sense removed